multiversx_sdk/entrypoints/entrypoints.py

```diff
@@ -94,21 +108,39 @@ def create_network_provider(self) -> Union[ApiNetworkProvider, ProxyNetworkProvi
     def create_delegation_controller(self) -> DelegationController:
         return DelegationController(self.chain_id, self.network_provider)
 
+    def create_delegation_factory(self) -> DelegationTransactionsFactory:
+        return DelegationTransactionsFactory(TransactionsFactoryConfig(self.chain_id))
+
     def create_account_controller(self) -> AccountController:
         return AccountController(self.chain_id)
 
+    def create_account_factory(self) -> AccountTransactionsFactory:
```

Comment:
Ambiguous naming. Maybe we should rename all functions to `create_..._transactions_factory()`?

Reply:
Indeed, renamed